apps/prairielearn/src/ee/pages/instructorAiGenerateDrafts/instructorAiGenerateDrafts.html.ts

```diff
@@ -113,8 +103,7 @@ export function InstructorAIGenerateDrafts({
 
             <div class="mb-3">
               <label class="form-label" for="user-prompt-llm">
-                Give a high-level overview of the question. What internal parameters need to be
-                generated and what information should be provided to students?
+                Give a high-level overview of the question.
```

Comment:
I'd like to say a little more here. How's this for a starter?

```suggestion
                Give a high-level overview of the question, including what parameters should be generated and how students should provide their response.
```

We can workshop this some more too.

Reply:
(Unresolving conversation for now so that @mwest1066 can chime in, I pinged him out of band)